vercel/src/lib/AsciiPreview.svelte

```diff
@@ -0,0 +1,82 @@
+<script lang="ts">
+  import Chip from './Chip.svelte';
+  import type { LanguageColors } from '../types';
+
+  export let name: string;
+  export let ansi: string[];
+  export let hex: string[] | null = null;
+  export let chip: string;
+  export let ascii: string = '';
+
+  let dark = true;
+  let trueColor = hex != null;
+
+  $: html = ascii
+    .split('\n')
+    .map((line) => {
+      // TODO Clean up, this is hard to read
+      let spanCount = 0;
+      const htmlLine = line.replace(/\{(\d+)\}/g, (_match, index) => {
+        const i = Number.parseInt(index, 10);
+        const spanText = trueColor
+          ? `<span style="color: ${hex[i]}">`
+          : `<span class="text-ansi-${ansi[i]}">`;
```

Comment:
```suggestion
          : `<span style="color: ${ansi[i]}">`;
```
I don't see the point of having an extra stylesheet for this (app.css).

Reply:
I guess I was just being a bit obsessive about the "use stylesheets instead of the style attribute" rule 😆